### PR TITLE
Add: System tests #5

### LINKED ISSUE #5: Add: System test for the command line
## Description

The system tests need to exist.

There should be several tests that test the most typical scenarios of usage, and some potential edge cases.

## Goals

- [x] File and folder structure
- [x] Plan tests
- [x] Create test data
- [x] Create expected data
- [x] Write tests

### PR DESCRIPTION
**Pull request by Gabriele-Gentile**
## Description

Added tests for most cross-cat flags.

## Issues

- Close: #5

### CODE REVIEW

File: test/system/system-test.ts [hunk 1]
```diff
@@ -6,22 +6,89 @@
 /**
  * Performs a series of system tests.
  */
-import { spawn, fork, exec } from "child_process";
+import { ChildProcessWithoutNullStreams, spawn } from "child_process";
+import { ok, strictEqual } from "assert";
+import { dinosaursShowTabs as expectedDinosaursShowTabs } from "../expected/index";
+import { fruits as expectedFruits } from "../expected/index";
```
Comment (winterhotlatte) [inline test/system/system-test.ts:12]: Codacy Issue found: ['/src/test/expected/index.ts' imported multiple times.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5179809)

**Comment by winterhotlatte** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5179809&bid=16999340)

File: test/system/system-test.ts [hunk 2]
```diff
@@ -6,22 +6,89 @@
 /**
  * Performs a series of system tests.
  */
-import { spawn, fork, exec } from "child_process";
+import { ChildProcessWithoutNullStreams, spawn } from "child_process";
+import { ok, strictEqual } from "assert";
+import { dinosaursShowTabs as expectedDinosaursShowTabs } from "../expected/index";
```
Comment (winterhotlatte) [inline test/system/system-test.ts:11]: Codacy Issue found: ['/src/test/expected/index.ts' imported multiple times.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5179809)

File: test/system/system-test.ts [hunk 3]
```diff
@@ -6,22 +6,89 @@
...
+import { fruits as expectedFruits } from "../expected/index";
+import { fruitsNumber as expectedFruitsNumber } from "../expected/index";
```
Comment (winterhotlatte) [inline test/system/system-test.ts:13]: Codacy Issue found: ['/src/test/expected/index.ts' imported multiple times.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5179809)

File: test/system/system-test.ts [hunk 4]
```diff
@@ -6,22 +6,89 @@
...
+import { fruitsShowEnds as expectedFruitShowEnds } from "../expected/index";
```
Comment (winterhotlatte) [inline test/system/system-test.ts:14]: Codacy Issue found: [Imports should be sorted alphabetically.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5179809)

File: test/system/system-test.ts [hunk 5]
```diff
@@ -6,22 +6,89 @@
...
+import { vegetablesNumberNonBlank as expectedVegetablesNumberNonBlank } from "../expected/index";
```
Comment (winterhotlatte) [inline test/system/system-test.ts:15]: Codacy Issue found: ['/src/test/expected/index.ts' imported multiple times.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5179809)

File: test/system/system-test.ts (hunk 4 above)
Comment (winterhotlatte) [inline test/system/system-test.ts:14]: Codacy Issue found: ['/src/test/expected/index.ts' imported multiple times.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5179809)

**Comment by winterhotlatte** [comment]
All good.
